### PR TITLE
Remove Callbacks to Show Entry or Error

### PR DESCRIPTION
**Pull request by philippfromme**
The callbacks will no longer be needed. Entries will be shown based on their ID.

---

Related to https://github.com/bpmn-io/internal-docs/issues/531

### CODE REVIEW

File: test/spec/provider/bpmn/MessageProps.spec.js [hunk 1]
```diff
@@ -259,38 +259,6 @@ describe('provider/bpmn - MessageProps', function() {
 
     });
 
-
-    describe('show error', function() {
```
Comment (pinussilvestrus) [inline test/spec/provider/bpmn/MessageProps.spec.js:263]: Wouldn't it still make sense to have such an integration test? Maybe not for all specific props, but a general one (e.g. in the BpmnPropertiesPanelRenderer tests)?

Comment (philippfromme) [reply test/spec/provider/bpmn/MessageProps.spec.js:263]: Alright, I'll try and add one.

**Review by pinussilvestrus — APPROVED** [review]
🎸

File: test/spec/provider/bpmn/MessageProps.spec.js (hunk 1 above)
Comment (pinussilvestrus) [reply test/spec/provider/bpmn/MessageProps.spec.js:263]: Thanks!

File: test/spec/BpmnPropertiesPanelRenderer.spec.js [hunk 2]
```diff
@@ -518,6 +518,41 @@ describe('<BpmnPropertiesPanelRenderer>', function() {
       expect(getHeaderName(propertiesContainer)).to.eql('start');
     });
 
+
+    it('should show error', async function() {
```
Comment (pinussilvestrus) [inline test/spec/BpmnPropertiesPanelRenderer.spec.js:522]: I was a bit too quick, this one fails for properties panel versions < 16 (cf. https://github.com/bpmn-io/bpmn-js-properties-panel/runs/7252180335?check_suite_focus=true#step:7:64).

We can use [our helpers](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/test/TestHelper.js#L169) for this

```suggestion
     withPropertiesPanel('>=0.16')('should show error', async function() {
```

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]

**Pull request merged**